### PR TITLE
media-gfx/gnome-photos: version bump to 43.0

### PR DESCRIPTION
**Pull request by Schievel1**
This says 
```
Overview of changes in 43.beta
===============================
 * Build with soup3, drops libgfbgraph as it is not compatible with soup3 / rest 1.0 - Bilal Elmoussaoui
```
in NEWS, so i guess this should depend on libsoup3, but it did not depend an libsoup2 before and I can not find libsoup anywhere in the code.

- [ ] needs https://github.com/gentoo/gentoo/pull/27383 to be merged first, as we need geocode-glib-2.0 here

Signed-off-by: Pascal Jaeger <pascal.jaeger@leimstift.de>

### CODE REVIEW

File: media-gfx/gnome-photos/gnome-photos-43.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+IUSE="test upnp-av"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=x11-libs/cairo-1.14.0
+	>=x11-libs/gdk-pixbuf-2.36.8:2
+	>=media-libs/gegl-0.4.0:0.4[cairo,raw]
+	>=sci-geosciences/geocode-glib-3.26.4-r200:2
+	>=media-libs/gexiv2-0.14.0
+	>=dev-libs/glib-2.62.0:2
```
Comment (mattst88) [inline media-gfx/gnome-photos/gnome-photos-43.0.ebuild:24]: Looks like some new dependencies were added right here:

````diff
+libportal_dep = dependency('libportal')
+libportal_gtk3_dep = dependency('libportal-gtk3')
````

Needs to depend on `dev-libs/libportal[gtk]`.

File: media-gfx/gnome-photos/gnome-photos-43.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	>=net-libs/gnome-online-accounts-3.8.0:=
+	gnome-base/gsettings-desktop-schemas
+	>=x11-libs/gtk+-3.22.16:3
+	>=dev-libs/libdazzle-3.26.0
+	>=gui-libs/libhandy-1.1.90:1=
+	virtual/jpeg:0
```
Comment (mattst88) [inline media-gfx/gnome-photos/gnome-photos-43.0.ebuild:30]: `pkgcheck` says



DeprecatedDep | DEPEND: deprecated dependency: virtual/jpeg:0
-- | --


We need to replace this with `media-libs/libjpeg-turbo:0=`.

File: media-gfx/gnome-photos/gnome-photos-43.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="test upnp-av"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=x11-libs/cairo-1.14.0
+	>=x11-libs/gdk-pixbuf-2.36.8:2
+	>=media-libs/gegl-0.4.0:0.4[cairo,raw]
+	>=sci-geosciences/geocode-glib-3.26.4-r200:2
```
Comment (mattst88) [inline media-gfx/gnome-photos/gnome-photos-43.0.ebuild:22]: Just `sci-geosciences/geocode-glib:2` is enough.

File: media-gfx/gnome-photos/metadata.xml [hunk 4]
```diff
@@ -5,9 +5,6 @@
 	<email>gnome@gentoo.org</email>
 	<name>Gentoo GNOME Desktop</name>
 </maintainer>
-<use>
-	<flag name="flickr">Ensure runtime Flickr support</flag>
-</use>
```
Comment (mattst88) [inline media-gfx/gnome-photos/metadata.xml:10]: Can't remove this yet. It's still present in the v42.0 ebuild. We'll drop it when we drop the v42.0 ebuild.

**Review by mattst88 — CHANGES_REQUESTED** [review]

File: media-gfx/gnome-photos/gnome-photos-43.0.ebuild (hunk 1 above)
Comment (Schievel1) [reply media-gfx/gnome-photos/gnome-photos-43.0.ebuild:24]: Isn't libportal a flatpak thing? We have -Dflatpak=false

**Comment by mattst88** [comment]
This looks good. I'd just like to fix https://bugs.gentoo.org/653178 first. Let me give that a try.

**Comment by mattst88** [comment]
I see merge requests upstream that have been open for months and one (https://gitlab.gnome.org/GNOME/gnome-photos/-/merge_requests/52) improves the situation but then it fails a different way:


````
1/3 gnome-photos / basic.py                     FAIL             0.33s   (exit status 133 or signal 5 SIGTRAP)
>>> GSETTINGS_SCHEMA_DIR=/home/mattst88/projects/GNOME/gnome-photos/data GSETTINGS_BACKEND=memory MALLOC_PERTURB_=122 LC_ALL=C /usr/bin/dbus-run-session --config-file /home/mattst88/projects/GNOME/gnome-photos/build/tests/test-session.conf /home/mattst88/projects/GNOME/gnome-photos/build/../tests/basic.py
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
stderr:

(basic.py:2055633): dbind-WARNING **: 22:16:19.729: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files

(basic.py:2055633): dbind-WARNING **: 22:16:19.739: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files

(basic.py:2055633): dbind-ERROR **: 22:16:19.739: AT-SPI: Couldn't connect to accessibility bus. Is at-spi-bus-launcher running?
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
````

@leio: do you have any idea about this failure? Do we need to somehow start `at-spi-bus-launcher`?

**Comment by leio** [comment]
Unfortunately I don't know that one. But sounds likely. Perhaps some other upstream projects have something in their test runner to take care of it?

**Pull request closed without merging**